src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="get"
+      method="#"
+      class="search_form"
+      data-qa="big"
+    >
+      <input
+        type="search"
+        name="big_search"
```

Comment:
use camelCase for name attributes
![Screenshot 2023-03-21 at 17 42 56](https://user-images.githubusercontent.com/6526149/226661701-e2b3bcbb-6fb4-48d9-a746-25314242b42c.png)